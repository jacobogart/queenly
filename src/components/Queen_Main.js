import React, { Component } from 'react';
import Queen_Main from './Queen_Main.js';
// * CSS imports
import '../css/Main_Info.css';

export default class Queen_Main extends Component {
  constructor(props) {
    super(props);
    this.state = {
    }
  }

  render() {
    return (
      <main className="Queen-Main">

      </main>
    )
  }
}