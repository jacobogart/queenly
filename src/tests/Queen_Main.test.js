import React from 'react';
import ReactDOM from 'react-dom';
import Queen_Main from '../components/Queen_Main.js';
import { shallow } from 'enzyme';

const [nameOfFunctionHere] = jest.fn();

describe('Queen_Main', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Queen_Main />
    );
  });
  it('should match snapshop', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('should have default state', () => {
    expect(wrapper.state()).toEqual({
      // Put state here
    });
  });
})