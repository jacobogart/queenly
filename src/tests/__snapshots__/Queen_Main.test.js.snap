// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Queen_Main should match snapshop 1`] = `
<main
  className="Queen-Main Main_Info"
>
  <article
    className="queen-content"
  >
    <h2>
      Aunty Chan
    </h2>
    <h4>
      Instagram Bio:
    </h4>
    <p
      className="bio"
    >
      ❤️🧡💛💚💙💜💙💚💛🧡❤️ Double-D list Celebrity📍Chicago Drag Queen and Opinionated Scum ❤️🧡💛💚💙💜💙💚💛🧡❤️ IMHO EP. 4 💋⬇️
    </p>
  </article>
  <aside
    className="queen-links social-links"
  >
    <a
      href="https://www.instagram.com/aunty_chan/?hl=en"
    >
      <i
        className="fab fa-instagram"
      />
    </a>
    <a
      href="https://www.facebook.com/auntycherrychan"
    >
      <i
        className="fab fa-facebook-square"
      />
    </a>
    <a
      href="https://twitter.com/aunty_chan?lang=en"
    >
      <i
        className="fab fa-twitter-square"
      />
    </a>
  </aside>
</main>
`;
